src/test/java/educationalproject/programmingstuff/test_data_prep/TestDataFactory.java

```diff
@@ -0,0 +1,33 @@
+package educationalproject.programmingstuff.test_data_prep;
+
+import educationalproject.programmingstuff.model.User;
+
+import java.util.List;
+
+public class TestDataFactory {
```

Comment:
if I'm not mistaken, lombok has a useful annotation for such classes @UtilityClass or smth like that

Reply:
Done